sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -850,12 +850,6 @@ class AnalysisErrorSuite extends AnalysisTest {
       LocalRelation(a))
     assertAnalysisError(plan2, "Accessing outer query column is not allowed in" :: Nil)
 
-    val plan3 = Filter(
-      Exists(Union(LocalRelation(b),
```

Comment:
We can change this to a set operation other than Union instead of deleting it.

Reply:
I plan to add support for the other set ops in the next PR.